### PR TITLE
Allow Stripe to send receipt emails

### PR DESCRIPTION
**Pull request by kitsuta**
Stripe has the ability to send emails, and we have the option turned on in Stripe -- however, we were never sending Stripe an email address to use, so it never sent receipts. We now, finally, send an email off to Stripe.

### CODE REVIEW

File: uber/utils.py [hunk 1]
```diff
@@ -224,6 +224,7 @@ def __init__(self, targets=(), amount=None, description=None):
 
         self.amount = amount or self.total_cost
         self.description = description or self.names
+        self.email = self.models[0].email if self.targets and self.models[0].email else ''
```
Comment (kitsuta) [inline uber/utils.py]: This is the only file with relevant changes. The other stuff was from a (already-merged) PR.

**Comment by binary1230** [comment]
Note: after this is merged, we should login to all three of our stripe accounts to check receipt status, was in there recently and I think it was disabled for 1 of the accounts

**Comment by RobRuana** [comment]
Also, we should probably add `email` to the `to_dict()` method:
```python
    def to_dict(self):
        return {
            'targets': self.targets,
            'amount': self.amount,
            'description': self.description,
            'email': self.email
        }
```

**Comment by kitsuta** [comment]
@RobRuana Good idea on unit-testing the charge class itself! No tests existed for it yet so I hadn't thought of it. I also did add the email to to_dict -- nice catch!

Could you quickly pull down this branch and test it on your local machine to make sure test payments go through? I'm having problems on my machine with Stripe timing out, but I don't think it's code-related.

**Pull request merged**